client/state/signup/progress/selectors.js

```diff
@@ -0,0 +1,11 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import { get } from 'lodash';
+
+export function getSignupProgress( state ) {
+	return get( state, 'signup.progress', [] );
```

Comment:
not that when repeatedly called without the information available this will always return a new array and thus trigger re-renders. feel free to create a static empty object and use that instead…

```js
const emptyArray = [];

export const getSignupProgress = state => get( state, 'signup.progress', emptyArray );
```

Reply:
Ah, I had no idea. Good call.